src/googleclouddebugger/python_breakpoint.py

```diff
@@ -253,15 +253,32 @@ def GetBreakpointId(self):
 
   def GetExpirationTime(self):
     """Computes the timestamp at which this breakpoint will expire."""
-    # TODO: Move this to a common method.
-    if '.' not in self.definition['createTime']:
+    return self.GetCreateTime() + self.expiration_period
+
+  def GetCreateTime(self):
+    if 'createTime' in self.definition:
+      return self.GetTimeFromRfc3339Str(self.definition['createTime'])
+    else:
+      return self.GetTimeFromUnixMsec(
+          self.definition.get('createTimeUnixMsec', 0))
+
+  def GetTimeFromRfc3339Str(self, rfc3339_str):
+    #TODO : Move this to a common method.
```

Comment:
I'm not actually sure what this TODO means at this point.  Are we handling rfc3339 elsewhere in the codebase?  If so, this might be a good time to do that refactor.